src/frontend/components/shared/Modal.tsx

```diff
@@ -162,6 +163,9 @@ class Modal extends Component<IModalProps, IModalState> {
     const { show, children, toggle, ROUTE } = this.props
     const { isNewlyMounted } = this.state
 
+    const widthStyle = this.props.isHome ?
```

Comment:
would prefer adding a prop that is consumed by styled components which toggles this style instead of inline styles